lib/logstash/filters/dns.rb

```diff
@@ -219,8 +222,9 @@ def resolve(event)
         end
       rescue Resolv::ResolvTimeout
         @failed_cache[raw] = true if @failed_cache
-        @logger.warn("DNS: timeout on resolving the hostname.",
+        @logger.debug("DNS: timeout on resolving the hostname.",
```

Comment:
Ok I think it is fair to say that changing the log level is not a breaking change, and we need to re-evalute in the future how we handle logging for event processing, so I'm fine leaving this as debug.